src/components/Contentful/MediaItemsCentered.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+	<section-with-background-classic
+		:background-content="sectionBackground"
+		:vertical-padding="verticalPadding"
+	>
+		<template #content>
+			<kv-page-container>
+				<kv-grid
+					:style="customGridStyles"
+					class="tw-grid-cols-12 lg:tw-grid-cols-10 lg:tw-gap-y-4
+						tw-justify-items-center tw-text-center tw-rounded tw-bg-white
+						tw-py-5 lg:tw-px-[177px]"
+				>
+					<span
+						v-for="(item, index) in contentfulMediaItems"
+						:key="index"
+						class="tw-flex tw-justify-items-center tw-items-center tw-mb-2"
+						:class="[itemClasses, {
+							'md:tw-col-span-12': index === 9
+						}]"
+					>
+						<kv-contentful-img
+							:contentful-src="item.file.url"
+							:alt="item.file.description"
+							:width="item.file.details.image.width"
```

Comment:
Not super urgent, but I'm concerned with this width and height coming from the Contentful item details, as those are the width and height of the originally uploaded image and it's encouraged to upload a very large/high fidelity item to Contentful (and then we properly size them in the code here).

Can we determine what the max width is of this container and use that for the width? Since I assume the aspect ratios are different that would mean removing the height.

Reply:
These images are actually all smaller than the max of the container (most are 93px wide) so I think we're ok for now. Def and important improvement to make for the next round.